### DISCUSSION TITLE
JST EH to PA (PAP-05V-S) cable

### DISCUSSION THREAD

**Discussion by luigitare** [revk/xESP32-Faikout#260] [Q&A] [answered]
I have a 'Perfera Floor Standing Unit' and I need a 'JST EH to PA (PAP-05V-S) cable' for it. I'm really not an electrician, is there anyone selling (ready-made) cables for that? Unfortunately, I can't find any online. Thank you.

**Comment by davea** [comment, marked as answer]
I believe that cable is sold under Daikin part number EKRS21. Available from [here](https://www.saturnsales.co.uk/Daikin-EKRS21-Wiring-Harness.html) or [here](https://aircon-online.co.uk/product/wire-harness-for-new-s21-port/).

This provides a male connection, and the Faikin module also has a male connection, so you'll need to bridge the gap with some jumper wires - something like [these](https://shop.pimoroni.com/products/jumper-jerky?variant=348491271) should do the trick.

It is possible to make your own cables by hand if you have all the relevant tools and components - @njh documented his success in [this thread](https://github.com/revk/ESP32-Faikin/discussions/49#discussioncomment-6387359).

**Comment by sam-m7** [comment]
There are also direct PA to Dupont/Jumper wires sold on Aliexpress and maybe where you life there are also local shops selling them. I've also searched but haven't found a direct EH to PA cable (besides the Daikin one, but this alone is too short for the Perfera floor mount, also I think you would need to change the pin order). Another alternative would be buying PA (to PA or single head) and EH (to EH or single head) cables and connecting the wires together (by soldering or the use of wire connectors).
